christfellowshipchurch/web-app-v2
# Updated Analytics Provider to Segment

### About
This PR updates our analytics to start using Segment. We updated the `AnalyticsProvider` component to start supporting `identify()` and `track()` from Segment to start tracking basic button clicks and user identities.

### Test Instructions
* open any page on our website and use the [Segment Event Tracker](https://chrome.google.com/webstore/detail/segment-event-tracker/hbanigoffkilibdakdmmlgefndpjmajl?hl=en) Chrome plugin to see any of the events tracked with Segment.

### Screenshots
<img width="1788" alt="image" src="https://user-images.githubusercontent.com/46049974/201714555-0567d3e1-240c-405c-abdb-977f9ee9372e.png">

### Closes Tickets
[CFDP-2325]


[CFDP-2325]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2325?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
